src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -8,11 +9,33 @@ interface InstructorData {
   name: string;
   email: string;
   institution: string;
-  status: string;
+  status: Status;
   joinLink?: string;
   message?: string;
 }
 
+/**
+ * Status of an instructor signup.
+ */
+export enum Status {
```

Comment:
Are you planning to use this status somewhere else, or only for signup - if not, was thinking that it should be renamed to something more specific/descriptive.

Reply:
Nope, it shouldn't be used anywhere else. I agree with your thought here. How about NewInstructorStatus?

Reply:
I suggest adding the the notion of signup to the name, since this is the status of an instructor signup, so maybe something along the lines of `NewInstructorSignupStatus`/`InstructorSignupStatus`/`InstructorAddStatus`

Reply:
Resolved; new is redundant so `InstructorSignupStatus` would suffice